torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -642,182 +655,184 @@ PyTorchModelLoader::getSymbolsMapping() {
-        {AdaptiveAvgPoolInputs::output_size}},
-       {{"aten::reshape"},
-        &PyTorchModelLoader::loadReshape,
-        {ReshapeInputs::shape}},
-       {{"aten::_convolution"},
-        &PyTorchModelLoader::loadConvolution,
-        {
-            ConvInputs::weights,
-            ConvInputs::bias,
-            ConvInputs::stride,
-            ConvInputs::padding,
-            ConvInputs::dilation,
-            ConvInputs::transposed,
-            ConvInputs::output_padding,
-            ConvInputs::groups,
-            ConvInputs::benchmark,
-            ConvInputs::deterministic,
-            ConvInputs::cudnn_enabled,
-        }},
-       {{"aten::batch_norm"},
-        &PyTorchModelLoader::loadBatchNorm,
-        {
-            BNInputs::weights,
-            BNInputs::bias,
-            BNInputs::running_mean,
-            BNInputs::running_var,
-            BNInputs::training,
-            BNInputs::momentum,
-            BNInputs::eps,
-            BNInputs::cuddnn_enabled,
-        }},
-       {{"aten::max_pool2d"},
-        &PyTorchModelLoader::loadMaxPool2d,
-        {
-            MaxPoolInputs::kernel_size,
-            MaxPoolInputs::stride,
-            MaxPoolInputs::padding,
-            MaxPoolInputs::dilation,
-            MaxPoolInputs::ceil_mode,
-        }},
-       {{"aten::avg_pool2d"},
-        &PyTorchModelLoader::loadAvgPool2d,
-        {
-            AvgPoolInputs::kernel_size,
-            AvgPoolInputs::stride,
-            AvgPoolInputs::padding,
-            AvgPoolInputs::ceil_mode,
-            AvgPoolInputs::count_include_pad,
-            AvgPoolInputs::divisor_override,
-        }},
-       {{"aten::matmul"}, &PyTorchModelLoader::loadMatMul, {}},
-       {{"aten::mm"}, &PyTorchModelLoader::loadMM, {}},
-       {{"aten::bmm"}, &PyTorchModelLoader::loadBmm, {}},
-       {{"aten::addmm"},
-        &PyTorchModelLoader::loadAddMM,
-        {
-            AddMMInputs::alpha,
-            AddMMInputs::beta,
-        }},
-       {{"aten::flatten"},
-        &PyTorchModelLoader::loadFlatten,
-        {
-            FlattenInputs::start_dim,
-            FlattenInputs::end_dim,
-        }},
-       {{"aten::prelu"},
-        &PyTorchModelLoader::loadPRelu,
-        {
-            PReluInputs::weight,
-        }},
-       {{"aten::slice"},
-        &PyTorchModelLoader::loadSlice,
-        {
-            SliceInputs::dim,
-            SliceInputs::start,
-            SliceInputs::end,
-            SliceInputs::step,
-        }},
-       {{"aten::softmax"},
-        &PyTorchModelLoader::loadSoftMax,
-        {
-            SoftMaxInputs::dim,
-            SoftMaxInputs::dtype,
-        }},
-       {{"aten::topk"},
-        &PyTorchModelLoader::loadTopK,
-        {
-            TopKInputs::k,
-            TopKInputs::dim,
-            TopKInputs::largest,
-            TopKInputs::sorted,
-        }},
-       {{"prim::ConstantChunk"}, &PyTorchModelLoader::loadConstantChunk, {}}});
```

Comment:
This is indented differently?

Reply:
It's corrected by formater automatically..